quic/s2n-quic-core/src/recovery/bbr/drain.rs

```diff
@@ -0,0 +1,59 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    random,
+    recovery::{
+        bbr,
+        bbr::{startup, BbrCongestionController},
+    },
+    time::Timestamp,
+};
+use num_rational::Ratio;
+use num_traits::One;
+
+//= https://tools.ietf.org/id/draft-cardwell-iccrg-bbr-congestion-control-02#4.3.2
+//# In Drain, BBR aims to quickly drain any queue created in Startup by switching to a
+//# pacing_gain well below 1.0, until any estimated queue has been drained. It uses a
+//# pacing_gain that is the inverse of the value used during Startup, chosen to try to
+//# drain the queue in one round
+pub(crate) const PACING_GAIN: Ratio<u64> = Ratio::new_raw(1, 2);
+
+//= https://tools.ietf.org/id/draft-cardwell-iccrg-bbr-congestion-control-02#4.3.2
+//# BBREnterDrain():
+//#     BBR.state = Drain
+//#     BBR.pacing_gain = 1/BBRStartupCwndGain  /* pace slowly */
+//#     BBR.cwnd_gain = BBRStartupCwndGain      /* maintain cwnd */
+pub(crate) const CWND_GAIN: Ratio<u64> = startup::CWND_GAIN;
```

Comment:
Why is this a constant and not actually updating it in the `enter_drain` function?

Reply:
The `bbr::State.cwnd_gain()` function uses this constant while in the `Drain` state. I felt this was less error prone then having to remember to update the value every time the state changes